var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -133,7 +133,7 @@ class PyNumpy(PythonPackage):
 
     # meson.build
     # https://docs.scipy.org/doc/scipy/dev/toolchain.html#compilers
-    conflicts("%gcc@:8.3", when="@1.26:", msg="NumPy requires GCC >= 8.4")
+    conflicts("%gcc@:8.3", when="@1.25.2:", msg="NumPy requires GCC >= 8.4")
```

Comment:
IIRC the minimum version required for 1.25.2 was 8.0 or 8.1. It's hard to find exactly, because pre-Meson this wasn't enforced I think. It may have worked with older versions, depending on C++ compiler and selection/enabling of SIMD support.

Did you get a new CI failure somewhere that caused this change?

Reply:
I've been doing my testing on SLES 15.4, and I was previously chasing a build failure that this change resolved.  I cannot reproduce the issue just now (i.e. py-numpy v1.25.2 builds just fine with system installed gcc@7.5.0 and a spack built gcc@8.3.0), so it must have been related to misconfigured compilers on my end.  I'll remove this change from the patch stack.